modules/learn/pages/clusters-and-availability/xdcr-viewing-conflict-logs.adoc

```diff
@@ -0,0 +1,348 @@
+= Viewing Conflict Logs
+:description: pass:q[View and access conflict logs from the conflict collection.]
+:page-toclevels: 3
+
+[abstract]
+{description}
+
+XDCR conflict logs are your application documents, so they're added in regular collections called xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#create-conflict-collection[Conflict Collection] designated by you.
+
+You can access, view, and process these documents programmatically, similar to your application documents, using the available utilities, features, and options.
+
+For example, you can use RBAC to manage access to conflict collections.
+You can also use Couchbase services and tools such as Eventing and Kafka Connect to process conflict log documents.
+
+For information about the Conflict Record Document, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record].
+
+[#using-sdk-sqlpp-for-conflict-logs]
+== Using Couchbase SDK with SQL++ for Conflict Logs
+
+The most common way to view these documents is by using the Couchbase SDK with SQL++.
```

Comment:
The most common way to view the conflict documents is by using SQL++.

Side note:  I recall a doc review where it said:
`++ is Antora syntax - replaced SQL++ with {sqlpp}`